pkg/advisory/create_test.go

```diff
@@ -164,15 +169,15 @@ func TestCreate(t *testing.T) {
 			name: "no events",
 			req: Request{
 				Package:         "brotli",
-				VulnerabilityID: "CVE-2023-1234",
+				VulnerabilityID: "CGA-xoxo-xoxo-xoxo",
 			},
 			wantErr: true,
 		},
 		{
 			name: "event type doesn't match data type",
 			req: Request{
 				Package:         "brotli",
-				VulnerabilityID: "CVE-2023-1234",
+				VulnerabilityID: "CGA-xoxo-xoxo-xoxo",
```

Comment:
These will trigger errors (expected, based on `wantErr: true`) but for the wrong reason, right? We don't want the VulnerabilityID to be set. I think it'd be good to remove these values from these test cases, and create a new test case specifically for expecting an error as a result of having a non-empty VulnerabilityID.